### PR TITLE
Limitation of password.

### PR DESCRIPTION
**Pull request by watari3**
I add description about limitation of password.
Could anyone make sure this description ?

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -24,11 +25,19 @@ This demo site contains mostly all known plugins ; i.e. the official ones + the
 
 root / root
 
-## 3. Docker image?
+## 3. What's limitation of password?
+
+GitBucket has a limitation of password as below.
+
+- Use alphanumeric character
```
Comment (takezoe) [inline FAQ.md]: `-`, `_` and `.` are also allowed in password.

Comment (watari3) [reply FAQ.md]: Modified again.
Could you confirm it ?

**Comment by takezoe** [comment]
@watari3 Perfect! Can I merge this branch into GitBucket Wiki? or Could you apply changes in this branch into GitBucket Wiki by youself?

**Comment by watari3** [comment]
Thanks @takezoe . I will apply changes in this branch into GitBucket Wiki by myself.

**Pull request merged**